setup/setup.py

```diff
@@ -110,8 +110,8 @@ def getReleaseSrc():
 elif "bdist_pdb" in sys.argv:
 
     #make pdb archive
-    pyVersion = ('2.7', '3.5', '3.6', '3.7', '3.8', '3.9')
-    platform = ('Win32', 'x64')
+    pyVersion = ('3.11',)
+    platform = ('x64',)

```

Comment:
Yep, this is for python 3.11 onwards, for 3.10 I had to change _PY_fopen to _PY_fopen_obj
Also had to use the real python nuget instead of the self published python3.x @ussrhero added himself to nuget, but now you can only build it one by one as nuget (for no good particular reason) doesn't allow using multiple version of the same package

Reply:
Maybe Ill change to use the include path from the installed python on the machine?